lib/runner.js

```diff
@@ -1,14 +1,17 @@
 'use strict';
 
 const { EventEmitter } = require('events');
-const { ConciseReporter } = require('./report');
+// const { ConciseReporter } = require('./report');
+const { TabReporter } = require('./report');
 
 class Runner extends EventEmitter {
 
   constructor(options) {
     super();
     this.options = Object.assign({ runTodo: false }, options);
-    this.reporter = new ConciseReporter();
+    // this.reporter = new ConciseReporter();
+    this.reporter = new TabReporter();
+    //this.reporter = new ConciseReporter();
```

Comment:
Ditto

Reply:
@JuliaGerasymenko  `ConciseReporter` should be the default. And which reporter will be used should be defined by options
```suggestion
     this.reported = options.reporter || new ConciseReporter();
```